src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -240,18 +300,28 @@ def save_instance_profile_permissions(self) -> str | None:
         if len(instance_profile_access) == 0:
             logger.warning("No Mapping Was Generated.")
             return None
-        return self._save(instance_profile_access)
-
-    def _overwrite_mapping(self, buffer) -> str:
         path = f"{self._folder}/aws_instance_profile_info.csv"
+        return self._save(instance_profile_access, path)
```

Comment:
rebase on main and `self._installation.save(...)`

Reply:
Done